src/ssl.c

```diff
@@ -15573,13 +15590,91 @@ int wolfSSL_ASN1_INTEGER_cmp(const WOLFSSL_ASN1_INTEGER* a,
     return 0;
 }
 
-
 long wolfSSL_ASN1_INTEGER_get(const WOLFSSL_ASN1_INTEGER* i)
```

Comment:
Since this function uses <stdint.h> functionality, it should be protected with "#ifndef NO_STDINT".